modules/codemod/lib/v13/index.ts

```diff
@@ -1,11 +1,15 @@
 import {Transform} from 'jscodeshift';
 import addPillLabel from './addPillLabel';
+import renameDubLogosImports from './renameDubLogosImports';
+import renameDubLogosReferences from './renameDubLogosReferences';
 
 const transform: Transform = (file, api, options) => {
   // These will run in order. If your transform depends on others, place yours after dependent transforms
   const fixes = [
     // add codemods here
     addPillLabel,
+    renameDubLogosImports,
+    renameDubLogosReferences,
```

Comment:
nice ok these are sequential

Reply:
Yeah so some of the specs for the references might be odd because they're supposed to run after imports are done.